### ISSUE TITLE
Crash in Android Autofill settings screen 

### ISSUE THREAD

**Issue by ByteHamster** [Kunzisoft/KeePassDX#44]
Hi, I am noticing a crash in version 2.5.0-beta6-libre

<img src="https://user-images.githubusercontent.com/5811634/37522162-632c9162-2923-11e8-94b7-c38ddb663563.jpg" width="300">

```
-16 14:04:19.475  6136  6136 E AndroidRuntime: FATAL EXCEPTION: main
03-16 14:04:19.475  6136  6136 E AndroidRuntime: Process: com.android.settings, PID: 6136
03-16 14:04:19.475  6136  6136 E AndroidRuntime: android.content.ActivityNotFoundException: Unable to find explicit activity class {com.kunzisoft.keepass.libre/com.keepassdroid.settings.SettingsAutofillActivity}; have you declared this activity in your AndroidManifest.xml?
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1940)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1618)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.Activity.startActivityForResult(Activity.java:4487)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.Activity.startActivityForResult(Activity.java:4445)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.Activity.startActivity(Activity.java:4806)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.Activity.startActivity(Activity.java:4774)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at com.android.settings.applications.defaultapps.DefaultAppPreferenceController.lambda$-com_android_settings_applications_defaultapps_DefaultAppPreferenceController_2847(DefaultAppPreferenceController.java:74)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at com.android.settings.applications.defaultapps.-$Lambda$1znk7tt4c3WlH31v4jn2klbHhJA.$m$0(Unknown Source:8)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at com.android.settings.applications.defaultapps.-$Lambda$1znk7tt4c3WlH31v4jn2klbHhJA.onGearClick(Unknown Source:0)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at com.android.settings.widget.GearPreference.onClick(GearPreference.java:71)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.view.View.performClick(View.java:6294)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.view.View$PerformClick.run(View.java:24774)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:790)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6494)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
03-16 14:04:19.475  6136  6136 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
03-16 14:04:19.478  1958 16943 W ActivityManager:   Force finishing activity com.android.settings/.SubSettings
03-16 14:04:19.487  1958  2008 I ActivityManager: Showing crash dialog for package com.android.settings u0
```

**Comment by J-Jamet** [comment]
Thx for the reply, it's a bad link when opening the Autofill settings, I'll solve the problem for the next version.

**Comment by J-Jamet** [comment]
Fixed in the new version 2.5.0.0beta7
